packages/client/hmi-client/src/components/llm/tera-jupyter-chat.vue

```diff
@@ -489,4 +501,11 @@ section {
 	height: calc(100% - 3.5rem);
 	overflow-y: auto;
 }
+.rerun-message {
+	display: flex;
+	background-color: var(--surface-warning);
+	justify-content: space-between;
+	align-items: center;
+	padding: 0.5rem;
```

Comment:
Here a list of all our CSS variables: https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L981
```suggestion
	padding: var(--gap-2);
```